### PR TITLE
Update of README.md for new xero instructions with new screenshots

### PR DESCRIPTION
**Pull request by kayliefz**

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
Hi - this is not including your new images - but rather pointing the readme to them in your repo - I think it makes more sense to bring the images into the main repo

**Comment by kayliefz** [comment]
As far as I can tell I don't have permissions to add the images to the main repo, hence the approach I took.

**Comment by eileenmcnaughton** [comment]
You should be able to include them in the PR? Are they in a commit?

File: README.md [hunk 1]
```diff
@@ -19,34 +19,42 @@ then you will have the extensions added to the site.
...
-You should now have a Xero tab in the civi menu. From here you can edit the xero settings. To do this 
-
-You need a Xero api key. 
+SETUP XERO KEYS
 
 Log into https://api.xero.com/Application?redirectCount=0
 
 Choose My Applications, Add application
-<img src='https://raw.githubusercontent.com/eileenmcnaughton/nz.co.fuzion.civixero/master/docs/images/create_application.png'>
+<img src='https://github.com/kayliefz/nz.co.fuzion.civixero/blob/master/docs/images/Add_new_App.png'>
```
Comment (eileenmcnaughton) [inline README.md]: We just need to fix this back now - I'll merge first because it will probably make it easier to do the follow up

**Pull request merged**